src/Runtime.Base/src/Runtime.Base.csproj

```diff
@@ -56,6 +56,7 @@
     <Compile Include="System\Runtime\EEType.Runtime.cs" />
     <Compile Include="System\Runtime\EETypePtr.cs" />
     <Compile Include="System\Runtime\ExceptionHandling.cs" />
+    <Compile Include="System\Runtime\ExceptionHandling.wasm.cs" />
```

Comment:
Include this for wasm build only?

Reply:
Should I do this for 
```
src\System.Private.CoreLib\src\System.Private.CoreLib.csproj
src\Test.CoreLib\src\Test.CoreLib.csproj
```
as well?

Reply:
I think so

Reply:
I didn't do System.Private.CoreLib.csproj as I hit
```
"E:\GitHub\corert\buildscripts\..\Tools\dotnetcli\dotnet.exe" msbuild /m /ConsoleLoggerParameters:ForceNoAlign "/p:IlcPath=E:\GitHub\corert\tests\\..\bin\Windows_NT.x64.Debug" "/p:Configuration=Debug" "/p:OSGroup=Windows_NT" "/p:Platform=x64" "/p:RepoLocalBuild=true" "/p:FrameworkLibPath=E:\GitHub\corert\tests\..\bin\Windows_NT.x64.Debug\lib" "/p:FrameworkObjPath=E:\GitHub\corert\tests\..\bin\obj\Windows_NT.x64.Debug\Framework"  /p:NativeCodeGen=wasm E:\GitHub\corert\tests\src\Simple\Generics\Generics.csproj

Microsoft (R) Build Engine version 15.9.20+g88f5fadfbe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Generating native code
  Expected type 'System.Runtime.EHClauseIterator' not found in module 'System.Private.CoreLib' ([S.P.CoreLib]System.RuntimeExceptionHelpers.RuntimeFailFast(RhFailFastReason,Exception,native int,native int))
```
during `build`, I guess that is shared from `Windows_NT.x64.Debug`.

Reply:
> during build, I guess that is shared from Windows_NT.x64.Debug.

Maybe now that we have a CI for WASM, we could stop running the "Windows CoreLib with WASM codegen" frankentest? (Launching `tests\runtest.cmd` is supposed to test Windows x64 RyuJIT and CppCodegen by default, but it will also do a build-only step for HelloWasm using Windows x64 CoreLib - this is pretty wrong, it kind of works, but it was bound to hit issues like this at some point.)

Stopping that build would avoid issues like this (making it so that wasm tests only run when testing wasm could be a separate pull request)

Reply:
Ok, will look at that.  Putting back to WIP in the meantime.  Thanks